packages/core/plop/component/component-ts.txt

```diff
@@ -15,7 +15,7 @@ export interface {{properCase componentName}}Props extends VibeComponentProps {
 }
 
 const {{properCase componentName}}: VibeComponent<{{properCase componentName}}Props, HTMLElement> = forwardRef(
-  ({ className, id, "data-testid": dataTestId, customProp = "defaultValue" }, ref) => {
+  ({ className, id, "data-testid": dataTestId, customProp = "defaultValue" }: {{properCase componentName}}Props, ref) => {
```

Comment:
add type for ref, same as for props.
let's default it to HTMLDivElement, and it would yell in case something is not a div, but saying HTMLElement is too generic

Reply:
Can you give an example of a signature?

Reply:
Ok got it, removing, thanks